docs/api-versioning.md

```diff
@@ -0,0 +1,19 @@
+# API Versioning
+
+## API versions
+
+### Version URL
+
+This API uses semver, and version names follows the `v{major}` format. You can find the versions currently available by using the version selector at the top of this page.
```

Comment:
Add link to https://semver.org/ for transparency, e.g.: "This API uses [semver](https://semver.org/), and (...)"